test_scenarios/send_data.py

```diff
@@ -36,12 +40,14 @@ def main(args):
 
     process_file(
         time_shift,
-        boto3.client('s3', region_name=args.region),
+        boto3.client('s3', config=Config(s3={'addressing_style': 'virtual'}),
+                     endpoint_url='https://s3' + panther_base_helpers.FIPS_SUFFIX if panther_base_helpers.FIPS_ENABLED else None),
         args.bucket_name,
         data.get('Logs', []),
         data.get('LogType', ''),
         data.get('Format', 'jsonl'))
 
+
```

Comment:
Bunch of formatting changes that my IDE put in. Let me know if you don't like them.